lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -555,6 +563,12 @@ instance DefineSeverity StakePoolLog where
         MsgStakeDistribution _ -> Info
         MsgStakePoolRegistration _ -> Info
         MsgRollingBackTo _ -> Info
+        MsgUsingRankingEpochConstants ec
+            | (ec ^. #totalRewards) == Quantity 0
+                -> Notice
```

Comment:
From the logging [guidelines](https://github.com/input-output-hk/cardano-wallet/wiki/Logging-Guidelines)
> NOTICE -> needs attention; something not progressing properly
> WARNING -> may continue into an error condition if continued

I am not sure this is really a warning. If `R` is zero, there will be no error. The pools will just be ordered arbitrarily/randomly. 

It might be expected that `R` is zero (… can be debated).

It might be fine that the total stake is zero, ignoring the fact that I think to blocks will be produced then. But that's not the concern of this module.

Reply:
Fair enough.